src/app/util/basic-types.h

```diff
@@ -27,6 +27,8 @@
 
 // Pull in NodeId
 #include <lib/core/NodeId.h>
+// Pull in GroupId
```

Comment:
do these comments (this and above) provide any value?  They seem to say:

```cpp
// Pull in Foo
#include <.../Foo.h>
```

Reply:
Removed